ViewModels/LoginModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Phytime.ViewModels
+{
+    public class LoginModel
+    {
+        private string _password;
+        [Required(ErrorMessage = "Email is empty")]
+        public string Email { get; set; }
```

Comment:
lets add regexp validation for email

Wrong: fsdfsdfsdf
Valid: fsdfdsf@fsdfsdf